fixture.py

```diff
@@ -1,221 +1,62 @@
 from bs4 import BeautifulSoup
-import requests
-import os
+from requests import get
 
-os.system('color')
+# The main array for the teams URL
+TEAMS = ["86/real-madrid", "360/manchester-united", "364/liverpool", "160/paris-saint-germain", "83/barcelona",
+         "367/tottenham-hotspur", "132/bayern-munich"]

```

Comment:
```suggestion
TEAMS = [
    "86/real-madrid",
    "360/manchester-united",
    "364/liverpool",
    "160/paris-saint-germain",
    "83/barcelona",
    "367/tottenham-hotspur",
    "132/bayern-munich",
]
```
Will make adding / removing teams in the future easier to review.

Reply:
FWIW, I incorporated this into my PR here: https://github.com/alonelysaber/Fixture-Calendar/pull/3/files